diffusion/callbacks/log_diffusion_images.py

```diff
@@ -19,7 +19,10 @@ class LogDiffusionImages(Callback):
 
     Args:
         prompts (List[str]): List of prompts to use for evaluation.
-        size (int, optional): Image size to use during generation. Default: ``256``.
+        size (int, Tuple[int, int], optional): Image size to use during generation.
+            If using a tuple, specify as (height, width).  Default: ``256``.
+        batch_size (int, optional): The batch size of the prompts passed to the generate function. If not specified,
```

Comment:
Would it be better to set the default batch size to be 1 if this isn't specified so an OOM due to misconfiguring is less likely?

Reply:
Sounds good, it shouldn't slow eval too much and is better than the run crashing